lib/engine/game/g_1835/game.rb

```diff
@@ -25,14 +25,18 @@ class Game < Game::Base
                         brown: '#7b352a')
 
         CURRENCY_FORMAT_STR = '%dM'
+        # game end current or, when the bank is empty
+        GAME_END_CHECK = { bank: :current_or }.freeze
```

Comment:
There's no bankrupcy check here, so the game will continue with zero players left? (See 1846 on how to handle multiple bankrupcies)

Can be fixed later.